torch/fx/graph.py

```diff
@@ -115,6 +140,7 @@ def _name(self, target: Target) -> str:
             if _is_magic(op):
                 op = op[2:-2]
         op = op.replace('.', '_')
+        op = op.replace('*', '')
```

Comment:
why?

Reply:
names need to be valid identifiers, but we were previously passing in '*args' to this function. I hit the bug refactoring around special handling for `get_param` operators which stripped this off, but only in the delegate.